vmdb/app/views/layouts/listnav/_container_replicator.html.haml

```diff
@@ -0,0 +1,23 @@
+- if @record.try(:name)
+  #accordion.panel-group
+    = patternfly_accordion_panel(truncate(@record.name, :length => truncate_length), true, "icon") do
+      = render(:partial => "layouts/quadicon",
+        :locals         => {:mode => :icon,
+          :item                   => @record,
+          :size                   => 72,
+          :typ                    => :listnav})
+
+    = patternfly_accordion_panel(_("Properties"), false, "container_replicator_prop") do
+      %ul.nav.nav-pills.nav-stacked
+        %li
+          = link_to(_('Summary'), {:action => 'show', :id => @record, :display => 'main'}, :title => _("Show Summary"))
+
+    = patternfly_accordion_panel(_("Relationships"), false, "container_replicator_rel") do
+      %ul.nav.nav-pills.nav-stacked
+
+        - if role_allows(:feature => "ems_replicator_show") && !@record.ext_management_system.nil?
+          %li
+            = link_to("#{ui_lookup(:table => "ems_replicator")}: #{@record.ext_management_system.name}",
+                {:controller => "ems_replicator", :action => 'show', :id => @record.ext_management_system.id.to_s},
```

Comment:
@abonas :controller => "ems_replicator" is causing the test failure(routing error), this should be changed to :controller => 'ems_container'